src/client/hud/layers/EventsDisplay.ts

```
@@ -242,40 +261,70 @@ export class EventsDisplay extends LitElement implements Controller {
-    // control-panel rather than as a scroll-list entry.
-    if (event.message === "events_display.received_gold_from_captured_ship") {
-      return;
-    }
-
-    let description: string = event.message;
-    if (event.message.startsWith("events_display.")) {
-      description = translateText(event.message, event.params ?? {});
+    if (event.playerID !== null && (!myPlayer || myPlayer.smallID() !== event.playerID)) return;
+    if (event.message === "events_display.received_gold_from_captured_ship") return;
+
+    let description = event.message.startsWith("events_display.")
```

ESLint

[error] 'description' is never reassigned. Use 'const' instead.

(prefer-const)